python/docs/source/examples/time_centric.ipynb

```diff
@@ -0,0 +1,339 @@
+    "\n",
+    "See [the Kaskada documentation](../guide/index) for lots more information."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "BJ2EE9mSGtGB",
+   "metadata": {
+    "id": "BJ2EE9mSGtGB"
+   },
+   "source": [
+    "## Kaskada Client Setup\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "37db47ba",
+   "metadata": {
+    "tags": [
+     "hide-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import kaskada as kd\n",
+    "kd.init_session()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "5b838eef",
+   "metadata": {},
+   "source": [
+    "## Example dataset\n",
+    "\n",
+    "For this demo, we'll use a very small example data set, which, for simplicity and portability of this demo notebook, we'll read from a string.\n",
+    "\n",
+    "```{note}\n",
+    "For simplicity, instead of a CSV file or other file format we read and then parse data from a CSV string.\n",
+    "You can load your own event data from many common sources, including Pandas DataFrames and Parquet files.\n",
+    "See {py:mod}`kaskada.sources` for more information on the available sources.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ba4bb6b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For demo simplicity, instead of a CSV file, we read and then parse data from a\n",
+    "# CSV string. Kaskadaa\n",
+    "event_data_string = '''\n",
+    "    event_id,event_at,entity_id,event_name,revenue\n",
+    "    ev_00001,2022-01-01 22:01:00,user_001,login,0\n",
+    "    ev_00002,2022-01-01 22:05:00,user_001,view_item,0\n",
+    "    ev_00003,2022-01-01 22:20:00,user_001,view_item,0\n",
+    "    ev_00004,2022-01-01 23:10:00,user_001,view_item,0\n",
+    "    ev_00005,2022-01-01 23:20:00,user_001,view_item,0\n",
+    "    ev_00006,2022-01-01 23:40:00,user_001,purchase,12.50\n",
+    "    ev_00007,2022-01-01 23:45:00,user_001,view_item,0\n",
+    "    ev_00008,2022-01-01 23:59:00,user_001,view_item,0\n",
+    "    ev_00009,2022-01-02 05:30:00,user_001,login,0\n",
+    "    ev_00010,2022-01-02 05:35:00,user_001,view_item,0\n",
+    "    ev_00011,2022-01-02 05:45:00,user_001,view_item,0\n",
+    "    ev_00012,2022-01-02 06:10:00,user_001,view_item,0\n",
+    "    ev_00013,2022-01-02 06:15:00,user_001,view_item,0\n",
+    "    ev_00014,2022-01-02 06:25:00,user_001,purchase,25\n",
+    "    ev_00015,2022-01-02 06:30:00,user_001,view_item,0\n",
+    "    ev_00016,2022-01-02 06:31:00,user_001,purchase,5.75\n",
+    "    ev_00017,2022-01-02 07:01:00,user_001,view_item,0\n",
+    "    ev_00018,2022-01-01 22:17:00,user_002,view_item,0\n",
+    "    ev_00019,2022-01-01 22:18:00,user_002,view_item,0\n",
+    "    ev_00020,2022-01-01 22:20:00,user_002,view_item,0\n",
+    "'''\n",
+    "\n",
+    "Events = kd.sources.CsvString(event_data_string,\n",
```

Comment:
Is the capitalization intentional? We've generally used lowercase for assigned names (which is probably more Pythonic).